contracts/Example3.sol

```diff
@@ -2,13 +2,16 @@ pragma solidity ^0.8.4;
 
 contract Example3 {
     uint256 marginPercentage = 30;
+
     function getOwnerMargin(uint256 amount)
-        public
-        view
-        returns (uint256 amountForSender, uint256 amountForOwner)
+    public
+    view
+    returns (uint256 amountForSender, uint256 amountForOwner)
     {
+    unchecked {
         amountForSender = (amount * (100 - marginPercentage)) / 100;
```

Comment:
Here the catch was to save `marginPercentage` into memory since it is being accessed twice